alarm/models.py

```diff
@@ -55,7 +55,7 @@ def penultimate_for_trade_pair(cls, trade_pair):
 
     @classmethod
     @transaction.atomic
-    def save_as_recent(cls, trade_pair, high_price, low_price):
+    def update_or_create_prices(cls, trade_pair, high_price, low_price):
```

Comment:
Bad naming, as method does not operate on prices, it operates on candles.
What's worse, it does not update anything, it creates and deletes.
You can remember of this case such as an anti-pattern.

Reply:
++